src/main/java/com/techsophy/tsf/account/service/InternalService.java

```diff
@@ -3,7 +3,7 @@
 import com.techsophy.tsf.account.dto.InternalUserFormDataSchema;
 import org.springframework.transaction.annotation.Transactional;
 
-public interface InternalUserFormDataService
+public interface InternalService
```

Comment:
please dont waste my time by committing unnecessary changes, if you are not clear on the comments, pls discuss with me directly.